frontend/pages/settings.tsx

```diff
@@ -87,7 +87,7 @@ const Settings = ({
     {
       name: 'applications',
       label: 'Applications',
-      content: <ApplicationsPage whartonapplications={whartonapplications} />,
+      content: <ApplicationsPage whartonapplications={whartonApplications} />,
```

Comment:
Could be appropriate to also rename the prop in ApplicationsPage to be camelcase for simplicity

Reply:
Made changes